services/src/main/java/com/chatbot/services/HangoutsMessageSender.java

```diff
@@ -0,0 +1,83 @@
+package com.chatbot.services;
+
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.HttpRequestInitializer;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.chat.v1.HangoutsChat;
+import com.google.api.services.chat.v1.model.ActionParameter;
+import com.google.api.services.chat.v1.model.Button;
+import com.google.api.services.chat.v1.model.Card;
+import com.google.api.services.chat.v1.model.FormAction;
+import com.google.api.services.chat.v1.model.Message;
+import com.google.api.services.chat.v1.model.OnClick;
+import com.google.api.services.chat.v1.model.Section;
+import com.google.api.services.chat.v1.model.TextButton;
+import com.google.api.services.chat.v1.model.WidgetMarkup;
+import com.google.auth.http.HttpCredentialsAdapter;
+import com.google.auth.oauth2.GoogleCredentials;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+// This class initialises the HangoutsChat and can thereafter be used
+@Component
+public class HangoutsMessageSender {
```

Comment:
1. abstract class MessageSender, and this as a child
2. Create subdirectory for message senders